models/health.go

```diff
@@ -66,10 +66,28 @@ type WorkloadHealth struct {
 // 	desired = 1, current = 10, available = 0 would means that a user scaled down a workload from 10 to 1
 //  but in the operaton 10 pods showed problems, so no pod is available/ready but user will see 10 pods under a workload
 type WorkloadStatus struct {
-	Name              string `json:"name"`
-	DesiredReplicas   int32  `json:"desiredReplicas"`
-	CurrentReplicas   int32  `json:"currentReplicas"`
-	AvailableReplicas int32  `json:"availableReplicas"`
+	Name              string        `json:"name"`
+	DesiredReplicas   int32         `json:"desiredReplicas"`
+	CurrentReplicas   int32         `json:"currentReplicas"`
+	AvailableReplicas int32         `json:"availableReplicas"`
+	ProxyStatus       []ProxyStatus `json:"proxyStatus"`
+}
+
+type ProxyStatuses string
```

Comment:
Please, don't model a primitive type with an abstraction.

Also, we have this in the pods

```
Status              string            `json:"status"`
```